include/experimental/__p2630_bits/strided_slice.hpp

```diff
@@ -30,7 +30,7 @@ namespace detail {
 }
 
 // Slice Specifier allowing for strides and compile time extent
-template <class OffsetType, class ExtentType, class StrideType>
+template <class OffsetType, class ExtentType, class StrideType = std::integral_constant<size_t,1>>
```

Comment:
This does not conform with the current working draft (for C++26), as `strided_slice` has no default template arguments.

https://eel.is/c++draft/mdspan.sub.strided.slice

Please contact your National Body regarding the potential for submitting a comment to change C++26.  The US National Body has already submitted its comments.